easybuild/easyconfigs/p/PCAngsd/PCAngsd-0.97-foss-2018a-Python-2.7.14.eb

```diff
@@ -0,0 +1,52 @@
+easyblock = 'Tarball'
+
+name = 'PCAngsd'
+version = '0.97'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://www.popgen.dk/software/index.php/PCAngsd'
+description = """PCAngsd, which estimates the covariance matrix for low depth 
+NGS data in an iterative procedure based on genotype likelihoods and is able 
+to perform multiple population genetic analyses in heterogeneous populations."""
+
+toolchain = {'name': 'foss', 'version': '2018a'}
+
+source_urls = ['https://github.com/Rosemeis/pcangsd/archive/v%(version)s/']
+sources = [SOURCE_TAR_GZ]
+patches = [
+    'PCAngsd-python-hashbang.patch',
+]
+checksums = [
+    '05507ccf70b6464656ad9a63bb4d43d771d72d633055a08e056b9feb91f5c60a', 
+    '8729dbda52a94aa61cbd48bc73cd15155831ce22aebc30d9b4e96ff12c12a482', # PCAngsd-python-hashbang.patch
+]
+
+dependencies = [
+    ('Python', '2.7.14'),
+    ('numba', '0.37.0', versionsuffix),
+]
+
+exts_defaultclass = 'PythonPackage'
+#exts_filter = ("python -c 'import %(ext_name)s'", '')
```

Comment:
This should be either uncommented, or be removed.

Reply:
Apologies. Should have seen this myself.